apps/gql/schema/resolvers/index.ts

```diff
@@ -1,12 +1,51 @@
-import { type Resolvers } from "../types.generated";
-import { term } from "./Query/SozlukQuery/term";
-import { terms } from "./Query/SozlukQuery/terms";
-import { user } from "./Query/user";
+import { DateResolver, DateTimeResolver } from "graphql-scalars";
+
+import { parse, stringify } from "@kampus/gql-utils/global-id";
+import { type User } from "@kampus/prisma";
+import { assertNever } from "@kampus/std";
+import { type Dictionary } from "@kampus/std/dictionary";
```

Comment:
should this type be exported from "@kampus/std" ?

Reply:
i thouht about it but since `@kampus/std` itself is gonna be lots of helpers/utils/modules (at least that's how i envision it), it's actually an umbrella package. so namespacing it to one level deeper, "/dictionary" for this example, should help us avoiding the namespace collisions. Think of it as importing from `lodash/groupBy` instead of `lodash`. Also helps the app to "tree shake" better (not even sure if it's the right term 😅 )